service/worker/scanner/scanner.go

```diff
@@ -111,6 +111,7 @@ func New(
 	if err != nil {
 		resource.GetLogger().Fatal("failed to initialize zap logger", tag.Error(err))
 	}
+	zapLogger.Info("Initializing new scanner")
```

Comment:
nit: most other places in the repo uses the internal `log.Logger` which wraps zap.